source/concurrent/Promise.ooc

```diff
@@ -33,6 +33,13 @@ Promise: abstract class {
 	start: static func (action: Func) -> This {
 		_ThreadPromise new(action)
 	}
+	createEmpty: static func -> This { _EmptyPromise new() }
```

Comment:
In the other project it is a static `empty` property : `empty := static This new()`